source/tutorial/dotnet.txt

```diff
@@ -1,8 +1,12 @@
 .. _dotnet-tutorial:
 
-=============
-MAUI Tutorial
-=============
+==============================================
+Tutorial: Atlas Device Sync for .NET with MAUI
```

Comment:
I opted for a modified version of our "Atlas Device SDK for <LANGUAGE>" construction. These tutorials are effectively SDK tutorials, but since they're in the App Services docs, I thought it might make sense to genericize them as Device Sync tutorials. This also lines up better with the TOC entries.